Dockerfile

```diff
@@ -1,19 +1,21 @@
-# Build
-# This is the latest gradle Docker image available on the client with JDK8 support
-FROM gradle:4.7.0-jdk8
-USER root
+FROM openjdk:8u171-alpine
+
+WORKDIR /usr/src/app
 
 # Make client repos available inside this image if necessary
 # And disable Gradle daemon
 ENV GRADLE_USER_HOME /opt
 COPY docker/gradle-config/ $GRADLE_USER_HOME/
 
+COPY gradle/ ./gradle
+COPY gradlew ./
+
 # Copy project sources
 COPY build.gradle.kts settings.gradle.kts ./
 COPY src ./src
 
 # Copy Data Access Layer project sources
 COPY dal ./dal
 
-ENTRYPOINT ["gradle", "--no-daemon"]
+ENTRYPOINT ["./gradlew", "--no-daemon"]
```

Comment:
Is it working on linux (in contained and in local linux too)?
I mean there no executable permissions commited into git repo for `./gradlew` executable:
```
$ git ls-files -s | grep gradlew
100644 cccdd3d517fc5249beaefa600691cf150f2fa3e6 0       gradlew
100644 f9553162f122c71b34635112e717c3e733b5b212 0       gradlew.bat
```
So, to make binary executable:
```
$ git update-index --chmod=+x gradlew
```
Result:
```
$ git ls-files -s | grep gradlew
100755 cccdd3d517fc5249beaefa600691cf150f2fa3e6 0       gradlew
100644 f9553162f122c71b34635112e717c3e733b5b212 0       gradlew.bat
```
commit:
``` git commit -m " make gradlew executable"

Reply:
Thanks!